core/templates/dev/head/pages/preferences/preferences.html

```diff
@@ -155,19 +155,29 @@
             <div class="col-lg-10 col-md-10 col-sm-10">
               <div class="checkbox">
                 <label>
-                  <input type="checkbox" ng-model="canReceiveEmailUpdates" ng-change="saveEmailPreferences(canReceiveEmailUpdates, canReceiveEditorRoleEmail, canReceiveFeedbackMessageEmail)">
+                  <input type="checkbox" ng-model="canReceiveEmailUpdates" ng-change="saveEmailPreferences(canReceiveEmailUpdates, canReceiveEditorRoleEmail, canReceiveFeedbackMessageEmail,
+                  canReceiveSubscriptionEmail)">
                   <span translate="I18N_PREFERENCES_EMAIL_RECEIVE_NEWS"></span>
                 </label>
               </div>
               <div class="checkbox">
                 <label>
-                  <input type="checkbox" ng-model="canReceiveEditorRoleEmail" ng-change="saveEmailPreferences(canReceiveEmailUpdates, canReceiveEditorRoleEmail, canReceiveFeedbackMessageEmail)">
+                  <input type="checkbox" ng-model="canReceiveEditorRoleEmail" ng-change="saveEmailPreferences(canReceiveEmailUpdates, canReceiveEditorRoleEmail, canReceiveFeedbackMessageEmail,
+                  canReceiveSubscriptionEmail)">
                   Receive emails when someone gives you edit rights to an exploration
                 </label>
               </div>
               <div class="checkbox">
                 <label>
-                  <input type="checkbox" ng-model="canReceiveFeedbackMessageEmail" ng-change="saveEmailPreferences(canReceiveEmailUpdates, canReceiveEditorRoleEmail, canReceiveFeedbackMessageEmail)">
+                  <input type="checkbox" ng-model="canReceiveSubscriptionEmail" ng-change="saveEmailPreferences(canReceiveEmailUpdates, canReceiveEditorRoleEmail, canReceiveFeedbackMessageEmail,
+                  canReceiveSubscriptionEmail)">
+                  Receive emails when a creator to whom you have subscribed, publishes an exploration
```

Comment:
Receive emails when a creator you've subscribed to publishes a new exploration

Also, don't forget to add this for i18n. Just adding to en.json (and maybe hi.json) is fine -- check that the English version shows up in the other languages. With luck, translatewiki will handle remaining translations prior to the next release.